### PR TITLE
DEVX-1561 adds config.env values to be replaced by pattern matching in tools

### PR DESCRIPTION
**Pull request by rspurgeon**
This upgrades the file to have the values provided by automation after https://github.com/confluentinc/common-tools/pull/362

### CODE REVIEW

File: utils/config.env [hunk 1]
```diff
@@ -1,6 +1,28 @@
...
+# The following values are provided by Confluent's
+# release automation process.  The values in these
+# names will be adopted in scripts and expressions
+# over time to construct other
+# values required by this repository
+#####################################################
+CONFLUENT=5.4.1-SNAPSHOT
+CONFLUENT_DOCKER_TAG=5.4.x-latest
+CONFLUENT_SHORT=5.4
+CONFLUENT_PREVIOUS=5.3.0
```
Comment (ybyzek) [inline utils/config.env]: Not a big deal but IIUC this should be `5.4.0`

**Review by ybyzek — APPROVED** [review]
Minor comment and LGTM.  Thank you!

File: utils/config.env (hunk 1 above)
Comment (andrewegel) [reply utils/config.env]: It will actually be an empty string if its a version that ends in a `.0`

Comment (rspurgeon) [reply utils/config.env]: So then what @ybyzek says is correct... if this is 5.4.1 then PREVIOUS is 5.4.0?

Comment (andrewegel) [reply utils/config.env]: Yes

**Pull request merged**